File: DeidUAB/src/deid/ahocorasick/trie/handler/DefaultEmitHandler.java
package deid.ahocorasick.trie.handler;

import java.util.ArrayList;
import java.util.List;

import deid.ahocorasick.trie.Emit;

public class DefaultEmitHandler implements EmitHandler {

    private List<Emit> emits = new ArrayList<>();

    @Override
    public void emit(Emit emit) {
        this.emits.add(emit);
    }

    public List<Emit> getEmits() {
        return this.emits;
    }

}

File: DeIdClient/DeIdClient/CS/Controller/ahocorasick/handler/DefaultEmitHandler.cs
using System.Collections;

namespace Aho
{
    public class DefaultEmitHandler : EmitHandler
    {

    private ArrayList emits = new ArrayList();


    public void emit(Emit emit)
    {
        this.emits.Add(emit);
    }

    public ArrayList getEmits()
    {
        return this.emits;
    }

}

}

   